pkg/apis/batch/validation/validation.go

```diff
@@ -298,15 +302,18 @@ func ValidateCronJobCreate(cronJob *batch.CronJob, opts apivalidation.PodValidat
 // ValidateCronJobUpdate validates an update to a CronJob and returns an ErrorList with any errors.
 func ValidateCronJobUpdate(job, oldJob *batch.CronJob, opts apivalidation.PodValidationOptions) field.ErrorList {
 	allErrs := apivalidation.ValidateObjectMetaUpdate(&job.ObjectMeta, &oldJob.ObjectMeta, field.NewPath("metadata"))
-	allErrs = append(allErrs, validateCronJobSpec(&job.Spec, &oldJob.Spec, field.NewPath("spec"), opts)...)
+	jobClone := job.DeepCopy()
+	apivalidation.CleanPodTemplateSpec(&jobClone.Spec.JobTemplate.Spec.Template)
```

Comment:
Here only the job spec needs to be deep copied.

Reply:
done